### DISCUSSION TITLE
TypeError: NetworkError when attempting to fetch resource.

### DISCUSSION THREAD

**Discussion by vKxni** [splinetool/react-spline#69] [Q&A]
When trying to export a character from Spline using the React code, I get the following error
`TypeError: NetworkError when attempting to fetch resource.` is there any reason for that?

**Comment by marcofugaro** [comment]
I think your connection is having problems. Could you share a [codesandbox](https://codesandbox.io/) with the error?

**Comment by vKxni** [comment]
> I think your connection is having problems. Could you share a [codesandbox](https://codesandbox.io/) with the error?

Codesandbox is works but I tried it in my dev environment (react, nextjs) and there I got the error. No chance getting that running on my main website

**Comment by marcofugaro** [comment]
Could you share a repo containing your local dev environment then?

**Reply by vKxni** [reply]
Yea where should I share it?

**Reply by marcofugaro** [reply]
You can upload the .zip here
